src/game/GameStateMenu.cpp

```diff
@@ -414,89 +350,39 @@ void GameStateMenu::positionElements() {
 * Function renders all assets to the screen
 * Changes the color of any assets that are selected
 * Calls helper function to position elements in the window
+*
+* Revisions:
+* Now renders solely with the Renderer instance. (Michael Goll / March 16, 2017)
 */
 void GameStateMenu::render() {
     //Only draw when not minimized
     if (!game.window.isMinimized()) {
 
+        screenRect = {ZERO, ZERO, game.window.getWidth(), game.window.getHeight()};
```

Comment:
ZERO doesnt help with readability but it doesn't particularly hinder it either, it's just clunky